src/Utilities/ProcessorArchitecture.cs

```diff
@@ -16,6 +16,10 @@ public static class ProcessorArchitecture
         /// </summary>
         public const string X86 = "x86";
         /// <summary>
+        /// Represents the 64-bit AMD64 processor architecture.
+        /// </summary>
+        public const string X64 = "x64";
```

Comment:
I don't think I like adding this since it's redundant with `AMD64`. Can it be avoided, perhaps by mapping x64->amd64 higher in the callstack?

Reply:
You mean modifying the switch case to be `"x64"   => Path.Combine(sdkToolsPath, "x64"),`? Or checking if the platform is `x64`, then pass `AMD64` into `GeneratePathToTool`?

Reply:
I was thinking the latter (around the `archToLookFor` thing).